packages/@sanity/vision/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Espen Hovlandsdal
```

Comment:
We should move the copyright to Sanity.io as maintaining it will be a team effort from now on